roles/astarte-finalizer/README.md

```diff
@@ -0,0 +1,38 @@
+Role Name
```

Comment:
Populate this with something meaningful (also, the License is not the correct one)